python/tvm/auto_scheduler/measure.py

```diff
@@ -63,6 +63,9 @@
 # We use 1e10 instead of sys.float_info.max for better readability in log
 MAX_FLOAT = 1e10
 
+class CustomBuildFunc:
```

Comment:
Suggest we use one class BuildFunc, which could store default / ndk / custom and so on.

Reply:
done